easybuild/easyblocks/c/cp2k.py

```diff
@@ -272,7 +272,7 @@ def configure_common(self):
             regflags = 'NOOPT'
 
         # make sure a MPI-2 able MPI lib is used
-        mpi2libs = ['impi', 'MVAPICH2', 'OpenMPI']
+        mpi2libs = ['impi', 'MVAPICH2', 'OpenMPI', 'MPICH2', 'MPICH']
```

Comment:
looks trivial, but.. .:-)

We should probably be using the `mpi_family()` function here, rather than relying on `get_software_root`.

@ocaisa: up for looking into that?

Also: do you have a matching easyconfig to test this change? It's pretty clear that this shouldn't break anything that works already.